package-manifest.toml

```diff
@@ -30,6 +30,14 @@ to = "/var/svc/manifest/site/nexus"
 [[package.omicron-nexus.paths]]
 from = "out/console-assets"
 to = "/var/nexus/static"
+# Note, we could just map the whole "out/certs" directory, but this ensures
+# both files exist.
+[[package.omicron-nexus.paths]]
+from = "out/certs/cert.pem"
+to = "/var/nexus/certs/cert.pem"
+[[package.omicron-nexus.paths]]
+from = "out/certs/key.pem"
+to = "/var/nexus/certs/key.pem"
```

Comment:
This might be a longer-term issue to resolve, but I'd like for this to not necessarily be bundled with the package.

However, this may require a different mechanism for acquiring / rotation of the certificate.

Reply:
I followed up today and met with Rick to discuss this.

My takeaways:
- RSS *must* provide a way for users to input the Nexus IP address + initial certificates
  - Later, an interface through Nexus will support modifying these values
- A key storage mechanism like https://github.com/hashicorp/vault - not necessarily this particular technology, but something highly available - should be used to store keys for both internal and externally facing services.
  - For **internal services**, we could concoct a scheme where:
    - When booted, services generate a public/private key pair
    - The private key stays private to the service while it exists in-memory
    - The public key gets published to "vault" - or whatever key management facility we use - and signed by the internal root CA. This should rely on trusting "who is launching the service" (namely, sled agent) to extend this trust.
  - For **external services**, we rely on user input for these certificates. We could store them in this "vault", but cannot generate them ourselves.

Reply:
This is tracked by https://github.com/oxidecomputer/omicron/issues/1528